File: content/examples/other/leetcode/problems/climbing-stairs/example-1/Solution.java
public class Solution {
  static int climbStairsNaive(int n) {
    if (n <= 2) return n;
    return climbStairsNaive(n - 1) + climbStairsNaive(n - 2);
  }

  public static void main(String[] args) {
    System.out.println(climbStairsNaive(5));
  }
}

File: content/examples/other/leetcode/problems/climbing-stairs/example-1/solution.js
function climbStairsNaive(n) {
  if (n <= 2) return n;
  return climbStairsNaive(n - 1) + climbStairsNaive(n - 2);
}

if (require.main === module) {
  console.log(climbStairsNaive(5));
}

module.exports = { climbStairsNaive };
